src/utils/review.ts

```diff
@@ -0,0 +1,40 @@
+import { AdminReview } from "@/services/api/admin/useReviews";
+
+function isGreaterThan24Hours(date: Date): boolean | null {
+  if (!date) return null;
+
+  const createdReview = new Date(date);
+  const now = new Date();
+  const differenceInMillis = now.getTime() - createdReview.getTime();
+
+  const millisecondsIn24Hours = 24 * 60 * 60 * 1000;
+
+  return differenceInMillis > millisecondsIn24Hours;
+}
+
+export const getReviewStatus = (review: AdminReview) => {
+  // checks if it is a review, NB: it is used in  the tableItems file
+  if (!review?.transcript?.status) {
+    return false;
+  }
+
+  const isExpired =
+    !review.mergedAt &&
+    review.archivedAt &&
+    (isGreaterThan24Hours(review.createdAt) ||
+      review.transcript.status?.toLowerCase() === "queued");
```

Comment:
an expired review might not been archived yet.
This logic doesn't cover that case, resulting to this below. Which is not bad, as it displays the review correctly under "Expired" but it assigns a wrong status.
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/18506343/561c3364-6405-46ac-91d8-04eb993a2991)

check the discussion here: https://github.com/bitcointranscripts/transcription-review-backend/pull/288#discussion_r1603201061

@Extheoisah couldn't we return statuses directly from backend? so that we don't need to have this kind of logic on the frontend?